### PR TITLE
Add e2e tests to travis CI

### LINKED ISSUE #72: e2e Tests Run Through Travis CI
## What happens?

e2e test do not currently run through travis CI.

## What were you expecting to happen?

e2e tests run through Travis

## Steps to reproduce: 

N/A

## Any errors, stacktrace, logs? 

N/A

## Environment:

* Runtime version(Java, Go, Python, etc): Go 1.12
* Desktop OS/version: N/A

## Additional comments:

N/A

### PR DESCRIPTION
**Pull request by etsauer**
# Pull Request Template

## Description

Adding minikube to travis and running e2e test script from Travis.

Fixes #72 

## Type of change

Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,17 +4,30 @@ go_import_path: github.com/KohlsTechnology/eunomia
...
   - travis_retry go mod vendor
   - go get golang.org/x/lint/golint
 
+before_script:
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.14.2/bin/linux/amd64/kubectl
+- sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+- curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+- sudo chmod +x minikube && sudo mv minikube /usr/local/bin/
+- curl -LO https://git.io/get_helm.sh && chmod 700 get_helm.sh && ./get_helm.sh
+- RELEASE_VERSION=v0.9.0; curl -OJL https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu && chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu && sudo mkdir -p /usr/local/bin/ && sudo cp operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk && rm operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
```
Comment (Smiley73) [inline .travis.yml]: Is there a more central place we can define versions in? Would make it easier for future maintenance.

Comment (etsauer) [reply .travis.yml]: @Smiley73 done.

File: .travis.yml [hunk 2]
```diff
@@ -4,17 +4,30 @@ go_import_path: github.com/KohlsTechnology/eunomia
 go:
 - "1.12"
 env:
-  - GO111MODULE=on
+  - GO111MODULE=on CHANGE_MINIKUBE_NONE_USER=true OPERATOR_SDK_VERSION=v0.9.0 KUBE_VERSION=v1.14.2
```
Comment (Smiley73) [inline .travis.yml]: Not familiar with travis, but is there any chance to have one ENV variable per line?

Comment (etsauer) [reply .travis.yml]: @Smiley73 specifying env vars on separate lines creates a "matrix" for test cases, passing different sets of env vars to test different versions, settings etc. This is how it should be set.

Comment (etsauer) [reply .travis.yml]: @Smiley73 actually, let do one refactor.

File: .travis.yml [hunk 3]
```diff
@@ -4,17 +4,34 @@ go_import_path: github.com/KohlsTechnology/eunomia
...
   - travis_retry go mod vendor
   - go get golang.org/x/lint/golint
 
+before_script:
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl
+- sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+- curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+- sudo chmod +x minikube && sudo mv minikube /usr/local/bin/
+- curl -LO https://git.io/get_helm.sh && chmod 700 get_helm.sh && ./get_helm.sh
+- curl -OJL https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu && chmod +x operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu && sudo mkdir -p /usr/local/bin/ && sudo cp operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk && rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
```
Comment (vinny-sabatini) [inline .travis.yml]: Since this line is quite long, can we break up the `&&`'s into just separate lines?

File: .travis.yml [hunk 4]
```diff
@@ -4,17 +4,34 @@ go_import_path: github.com/KohlsTechnology/eunomia
 go:
 - "1.12"
 env:
+  global:
   - GO111MODULE=on
+  - CHANGE_MINIKUBE_NONE_USER=true
+  matrix:
+  - OPERATOR_SDK_VERSION=v0.9.0 KUBE_VERSION=v1.14.2
```
Comment (seanmalloy) [inline .travis.yml]: Please change to SDK Version v0.8.1.

File: .travis.yml (hunk 1 above)
Comment (Smiley73) [reply .travis.yml]: @etsauer I do not see the changes...

File: .travis.yml (hunk 3 above)
Comment (seanmalloy) [reply .travis.yml]: @etsauer fixed it!

**Pull request merged**